src/fast_pbkdf2/.gitignore

```diff
@@ -0,0 +1,30 @@
+.rebar3
```

Comment:
Noticed from rebar compiler, it leaves the `src/fast_pbkdf2/compile_commands.json` behind as well. Can probably add it to .gitignore. Either this one or the top level one.

```
% git status

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	src/fast_pbkdf2/compile_commands.json
```